test/uk/gov/hmrc/agentaccesscontrol/helpers/UnitTest.scala

```diff
@@ -14,27 +14,15 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.agentaccesscontrol.support
+package uk.gov.hmrc.agentaccesscontrol.helpers
 
-import org.mockito.Mockito
-import org.scalatestplus.mockito.MockitoSugar
-import org.scalatest.{BeforeAndAfterEach, Suite}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
 
-import scala.reflect.Manifest
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Awaitable}
 
-trait ResettingMockitoSugar extends MockitoSugar with BeforeAndAfterEach {
-  this: Suite =>
-
-  var mocksToReset = Seq.empty[Any]
-
-  def resettingMock[T <: AnyRef](implicit manifest: Manifest[T]): T = {
-    val m = mock[T](manifest)
-    mocksToReset = mocksToReset :+ m
-    m
-  }
-
-  override protected def beforeEach(): Unit = {
-    super.beforeEach()
-    Mockito.reset(mocksToReset: _*)
-  }
+trait UnitTest extends AnyWordSpec with Matchers {
```

Comment:
PlaySpec extends AnyWordSpec and (must)Matchers which may make this trait unnecessary